src/styles/_mixins.scss

```diff
@@ -0,0 +1,78 @@
+@mixin link {
+  color: $company-green;
+  font-size: inherit;
+  font-family: $main-font;
+  background-color: transparent;
+  border: none;
+  outline: none;
+  text-decoration: underline;
```

Comment:
text-decoration property defined two times and will be overwritten to 'none'

Reply:
Fixed